docs/extensibility.md

```diff
@@ -29,531 +28,630 @@ pub trait DiscoveryHandler {
+Now you are ready to **build the HTTP broker**!  To do so, we simply need to run this step from the base folder of the Akri repo:
 
-```sh
-docker build -t nessie:extensibility -f samples/brokers/nessie/Dockerfile .
-```
+```bash
+HOST="ghcr.io"
+USER=[[GITHUB-USER]]
+BROKER="http-broker"
+TAGS="v1"
 
-Having built the nessie container, in order to use it in a cluster, you need to **push the nessie broker** to a container repo:
+IMAGE="${HOST}/${USER}/${BROKER}:${TAGS}"
 
-```sh
-# Log into your container repo ... in this case, ghcr using your Github username
-# and a Github PAT created to access ghcr
-echo <GITHUB PAT> | docker login -u <GITHUB USERNAME> ghcr.io --password-stdin
-# Create a container tag corresponding to your container repo
-docker tag nessie:extensibility ghcr.io/<GITHUB USERNAME>/nessie:extensibility
-# Push the nessie container to your container repo
-docker push ghcr.io/<GITHUB USERNAME>/nessie:extensibility
+docker build \
+--tag=${IMAGE} \
+--file=./samples/brokers/http/Dockerfiles/standalone \
+. && \
+docker push ${IMAGE}
 ```
 
-### Create a new Configuration
-Once the nessie broker has been created (assuming `ghcr.io/<GITHUB USERNAME>/nessie:extensibility`), the next question is how to deploy it.  For this, we need to create a Configuration called `nessie.yaml` that leverages our new protocol.  
-
-Please update the yaml below to:
-* Specify a value for the imagePullSecrets. This can be any name and will correspond to a Kubernetes secret you create, which will contain your container repo credentials.  Make note of the name you choose, as this will be used later in `kubectl create secret` and `helm install` commands.
-* Specify a value for your container image that corresponds to the container repo you are using
-
+To deploy the standalone broker, we'll need to create an Akri Configuration `./samples/brokers/http/kubernetes/http.yaml` (be sure to update **image**):
 ```yaml
 apiVersion: akri.sh/v0
 kind: Configuration
 metadata:
-  name: nessie
+  name: http
 spec:
   protocol:
-    nessie:
-      nessieUrl: https://www.lochness.co.uk/livecam/img/lochness.jpg
-  capacity: 5
+    http:
+      discoveryEndpoint: http://discovery:9999/discovery
+  capacity: 1
   brokerPodSpec:
-    hostNetwork: true
-    imagePullSecrets:
-    - name: <SECRET NAME>
+    imagePullSecrets: # Container Registry secret
+      - name: SECRET
     containers:
-    - name: nessie-broker
-      image: "ghcr.io/<GITHUB USERNAME>/nessie:extensibility"
-      resources:
-        limits:
-          "{{PLACEHOLDER}}" : "1"
-  instanceServiceSpec:
-    ports:
-    - name: grpc
-      port: 80
-      targetPort: 8083
-  configurationServiceSpec:
-    ports:
-    - name: grpc
-      port: 80
-      targetPort: 8083
+      - name: http-broker
+        image: IMAGE
+        resources:
+          limits:
+            "{{PLACEHOLDER}}": "1"
+```
+
+> **NOTE** If you're using a non-public repo, you can create an `imagePullSecrets` to authenticate

```

Comment:
Since the configuration is using image pull secrets, maybe change this note into a clarification of why image pull secrets were used

Reply:
will make a note about creating a secret at the top of the doc and assume a value throughout.